fastify.js

```diff
@@ -40,12 +42,14 @@ function build (options) {
   }
 
   var log
+  var isAbstract = false
```

Comment:
can you make this variable name more precise?  `hasLogger` is a good one for example.

Reply:
done